packages/shorebird_cli/lib/src/compute.dart

```diff
@@ -0,0 +1,41 @@
+import 'dart:async';
+import 'dart:isolate';
+
+/// Perform [computation] with [input] in an [Isolate].
+Future<R> compute<R, M>(FutureOr<R> Function(M) computation, M input) async {
```

Comment:
I thought there was some fancy new dart support for this?  I guess that's coming in Dart 3 and not yet in Dart 2.x.

Reply:
Not that I'm aware of but it would be great to delete all this code. There's https://pub.dev/packages/compute but I didn't want to pull in another dependency just for this 🤷

Reply:
[Isolate.run](https://api.flutter.dev/flutter/dart-isolate/Isolate/run.html) has been [introduced in 2.19](https://api.flutter.dev/flutter/dart-isolate/Isolate/run.html).